django/utils/translation/trans_real.py

```diff
@@ -597,26 +599,30 @@ def join_tokens(tokens, trim=False):
                 if endbmatch:
                     if inplural:
                         if message_context:
-                            out.write(' npgettext(%r, %r, %r,count) ' % (
+                            out.write(' npgettext({p}{!r}, {p}{!r}, {p}{!r},count) '.format(
                                 message_context,
                                 join_tokens(singular, trimmed),
-                                join_tokens(plural, trimmed)))
+                                join_tokens(plural, trimmed),
+                                p=raw_prefix))
```

Comment:
care to put the closing parentheses on the next line to follow style we use elsewhere?